Drop-In/src/main/java/com/braintreepayments/api/VaultedPaymentMethodSelectedListener.java

```diff
@@ -0,0 +1,5 @@
+package com.braintreepayments.api;
+
+public interface VaultedPaymentMethodSelectedListener {
```

Comment:
Should this be `VaultedPaymentMethodSelectedCallback` instead of `Listener`?

Reply:
We decided to go with listener here since its invocation is directly tied to a `View.OnClickListener`.